controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -83,65 +149,7 @@ func NewKvUiPluginDeplymnt(hc *hcov1beta1.HyperConverged) (*appsv1.Deployment, e
 						"app": kvUIPluginDeploymentName,
 					},
 				},
-				Spec: corev1.PodSpec{
-					ServiceAccountName: "default",
-					SecurityContext:    components.GetStdPodSecurityContext(),
-					Containers: []corev1.Container{
-						{
-							Name:            kvUIPluginDeploymentName,
-							Image:           kvUiPluginImage,
-							ImagePullPolicy: corev1.PullIfNotPresent,
-							Resources: corev1.ResourceRequirements{
-								Requests: map[corev1.ResourceName]resource.Quantity{
-									corev1.ResourceCPU:    resource.MustParse("10m"),
-									corev1.ResourceMemory: resource.MustParse("100Mi"),
-								},
-							},
-							Ports: []corev1.ContainerPort{{
-								ContainerPort: hcoutil.UiPluginServerPort,
-								Protocol:      corev1.ProtocolTCP,
-							}},
-							SecurityContext:          components.GetStdContainerSecurityContext(),
-							TerminationMessagePath:   corev1.TerminationMessagePathDefault,
-							TerminationMessagePolicy: corev1.TerminationMessageReadFile,
-							VolumeMounts: []corev1.VolumeMount{
-								{
-									Name:      kvServingCertName,
-									MountPath: "/var/serving-cert",
-									ReadOnly:  true,
-								},
-								{
-									Name:      nginxConfigMapName,
-									MountPath: "/etc/nginx/nginx.conf",
-									SubPath:   "nginx.conf",
-									ReadOnly:  true,
-								},
-							},
-						},
-					},
-					PriorityClassName: "kubevirt-cluster-critical",
-					Volumes: []corev1.Volume{
-						{
-							Name: kvServingCertName,
-							VolumeSource: corev1.VolumeSource{
-								Secret: &corev1.SecretVolumeSource{
-									SecretName:  kvServingCertName,
-									DefaultMode: pointer.Int32(420),
-								},
-							},
-						},
-						{
-							Name: nginxConfigMapName,
-							VolumeSource: corev1.VolumeSource{
-								ConfigMap: &corev1.ConfigMapVolumeSource{
-									LocalObjectReference: corev1.LocalObjectReference{
-										Name: nginxConfigMapName,
-									},
-								},
-							},
-						},
-					},
-				},
+				Spec: podSpec,
```

Comment:
can you please keep the pod spec inline to make the change more compact?

Reply:
Sure. Done.